lib/functions.js

```diff
@@ -420,16 +422,24 @@ function launch_modal(title, href, init_modal, id) {
         id = '#bs-popup';
     }
 
+
     if (init_modal == null) {
         init_modal = 1;
     }
 
     $('html').addClass('modal-open');
 
     $(id).on('hidden.bs.modal', function () {
-        $('html').removeClass('modal-open');
-        $(this).html('');
-        $(this).data('modal', null);
+
+        if ($('.modal-backdrop').length < 1 ) {
+            console.log('mydebug->functions->launch_modal->remove_modal');
```

Comment:
Il console.log sarebbe da rimuovere, considerando che mostra informazioni utili esclusivamente a fini di debug.

Reply:
Come procedo, chiudo il pull request e ne genero un altro?

Reply:
Basterebbe fare un nuovo commit, come i precedenti, in cui rimuovi la riga:
console.log('mydebug->functions->launch_modal->remove_modal');